src/TestProblem/Hydro/Zeldovich/Init_TestProb_Zeldovich.cpp

```diff
@@ -260,27 +282,29 @@ double Zeldovich_coord_transformation_solver( const double x_shift_input, const
       x_Lagrangian_Trial = gsl_root_fsolver_root( s );
       x_lower_bound = gsl_root_fsolver_x_lower( s );
       x_upper_bound = gsl_root_fsolver_x_upper( s );
-      status = gsl_root_test_interval( x_lower_bound, x_upper_bound, Pert_Wave_Len/1.0e5, 1.0e-5 );
+      status = gsl_root_test_interval( x_lower_bound, x_upper_bound, Pert_Wave_Len/1.0e8, 1.0e-8 );
```

Comment:
Does lowering the tolerated error even further help improve the accuracy? You should adopt a reasonable minimum threshold such that the error convergence recorded in `Record__L1Err` does not depend on the threshold set here.